src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -13,14 +14,12 @@
 public class UserController {
 
     @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity<UserDto> addUser(@RequestBody UserDto userDto) {
-       userDto = new UserDto(3L, "user3", 1,45454);
-        return ResponseEntity.ok(userDto);
+    public ResponseEntity<UserAddDto> addUser(@RequestBody UserAddDto userAddDto) {
+        return ResponseEntity.ok(userAddDto);
     }
```

Comment:
Maybe void. It shouldn't return password to frontend

Reply:
It can. Actually it might returning UserDto (not UserAddDto). It all depends on DbService implementation. Output is same as input just for testing purposes.